mlrun/platforms/iguazio.py

```diff
@@ -99,14 +100,14 @@ def _mount_v3io_extended(task):
             )
 
         if not secret:
-            task = v3io_cred(access_key=access_key)(task)
+            task = v3io_cred(access_key=access_key, user=user)(task)
```

Comment:
There are very thorough tests for the whole mounting logic in `test_iguazio.py` I believe you can easily add test for the cases you fixed here

Reply:
Added.